docs/sql-migration-guide.md

```diff
@@ -87,7 +87,7 @@ license: |
 
   - In Spark version 2.4, when a spark session is created via `cloneSession()`, the newly created spark session inherits its configuration from its parent `SparkContext` even though the same configuration may exist with a different value in its parent spark session. Since Spark 3.0, the configurations of a parent `SparkSession` have a higher precedence over the parent `SparkContext`. The old behavior can be restored by setting `spark.sql.legacy.sessionInitWithConfigDefaults` to `true`.
 
-  - Since Spark 3.0, parquet logical type `TIMESTAMP_MICROS` is used by default while saving `TIMESTAMP` columns. In Spark version 2.4 and earlier, `TIMESTAMP` columns are saved as `INT96` in parquet files. To set `INT96` to `spark.sql.parquet.outputTimestampType` restores the previous behavior.
+  - Since Spark 3.0, parquet logical type `TIMESTAMP_MICROS` is used by default while saving `TIMESTAMP` columns. In Spark version 2.4 and earlier, `TIMESTAMP` columns are saved as `INT96` in parquet files. Note that, some SQL systems like Impala 2.x can only read `INT96` timestamps, you can set `spark.sql.parquet.outputTimestampType` as `INT96` to restore the previous behavior and keep interoperability.
```

Comment:
I think you guard yourself so much. Impala has already supported the same as Spark:
https://github.com/cloudera/Impala/blob/63ff49de748692758bc465b3787c4193de78936c/be/src/exec/parquet/parquet-common.h#L489-L501

And I am not sure that we should advertise other systems in Spark docs.

Reply:
Parquet is an open format and interoperability is important. I know Impala supports the logical timestamp since 3.2.0, that why I use "Impala 2.x" here.

Reply:
Shouldn't we maybe mention Hive first rather than Impala?